README.md

```diff
@@ -1,112 +1,204 @@
-# Node.js Long-term Support Working Group
+# Node.js Long Term Support Working Group
 
-# LTS schedule
+# Active Release Line Schedule
 
-<table>
+<table width="100%">
 <tr>
-  <th>LTS Status</th>
-  <th>Release</th>
-  <th>LTS Start</th>
-  <th>Maintenance Start</th>
-  <th>LTS End</th>
+  <th width="10%">Release Line</th>
+  <th width="30%">Current Status</th>
+  <th width="20%">LTS Start</th>
+  <th width="20%">Maintenance Start</th>
+  <th width="20%">LTS End</th>
 </tr>
+
+
 <tr>
-  <td><b>Maintenance</b></td>
-  <td>v0.10</td>
-  <td>-</td>
-  <td>2015-10-01</td>
-  <td>2016-10-01</td>
+  <th>v0.10</th>
+  <td>Maintenance</td>
+  <td align="center">-</td>
+  <td align="center">2015-10-01</td>
+  <td align="center">2016-10-01</td>
 </tr>
+
 <tr>
-  <td><b>Maintenance</b></td>
-  <td>v0.12</td>
-  <td>-</td>
-  <td>2016-04-01</td>
-  <td>2016-12-31</td>
+  <th>v0.12</th>
+  <td>Maintenance</td>
+  <td align="center">-</td>
+  <td align="center">2016-04-01</td>
+  <td align="center">2016-12-31</td>
 </tr>
+
 <tr>
-  <td><b>Active</b></td>
-  <td>v4</td>
-  <td>2015-10-01</td>
-  <td>2017-04-01</td>
-  <td>2018-04-01</td>
+  <th>v4</th>
+  <td>Active&nbsp;LTS&nbsp;("Argon")</td>
+  <td align="center">2015-10-01</td>
+  <td align="center">2017-04-01</td>
+  <td align="center">2018-04-01</td>
 </tr>
+
 <tr>
-  <td>N/A</td>
-  <td>v5</td>
+  <th>v5</th>
+  <td>Current&nbsp;(non-LTS)</td>
```

Comment:
I'm OK with this but I was being vague about it because so far we've resorted to leaving a label off v5 after we got v6. Having it explicitly called "Maintenance" would probably be a step up from that and clarify how it works.